modules/administration-guide/partials/proc_collecting-dev-workspace-operator-metrics-with-prometheus.adoc

```diff
@@ -86,24 +73,36 @@ data:
           tls_config:
             insecure_skip_verify: true
           static_configs:
-            - targets: ['devworkspace-webhookserver:9443'] <5>
+            - targets: ['devworkspace-webhookserver.__<DWO_{orch-namespace}>__:9443'] <5>
 ----
 <1> The rate at which a target is scraped.
 <2> The rate at which the recording and alerting rules are re-checked.
 <3> The resources that Prometheus monitors. In the default configuration, two jobs, `DevWorkspace` and `DevWorkspace webhooks`, scrape the time series data exposed by the `devworkspace-controller-metrics` and `devworkspace-webhookserver` Services.
-<4> The scrape metrics from port `8443`.
-<5> The scrape metrics from port `9443`.
+<4> Scrape metrics from port `8443`. Replace `__<DWO_namespace>__` with the {orch-namespace} where the `devworkspace-controller-metrics` `Service` is located.
+<5> Scrape metrics from port `9443`. Replace `__<DWO_namespace>__` with the {orch-namespace} where the `devworkspace-webhookserver` `Service` is located.
 ====
 
+. Scale down and scale up the `Prometheus` Deployment to read the updated ConfigMap from the previous step.
++
+[subs="+attributes,quotes"]
+----
+$ {orch-cli} scale --replicas=0 deployment/prometheus -n monitoring && {orch-cli} scale --replicas=1 deployment/prometheus -n monitoring
+----
+
 .Verification steps
 
+. Use port forwarding to access the `Prometheus` Service locally:
++
+[subs="+attributes,quotes"]
+----
+$ {orch-cli} port-forward svc/prometheus 9090:9090 -n monitoring
+----
+. Verify that all targets are up by viewing the targets endpoint at `localhost:9090/targets`.
 . Use the Prometheus console to view and query metrics:
-** View the metrics at `http://__<prometheus_url>__/metrics`.
-** Query metrics from `http://__<prometheus_url>__/graph`.
+** View the metrics at `localhost:9090/metrics`.
+** Query metrics from `localhost:9090/graph`.
```

Comment:
If this makes sense, consider having `the` in both bullet points or in neither.

Reply:
In my latest commit, took the same approach as https://github.com/eclipse-che/che-docs/pull/2379#discussion_r905344015